config.lua

```diff
@@ -1,5 +1,8 @@
 Config = {}
 
+-- Key System Settings
+Config.PersistentKeys = true -- Whether the own keys should save into database or not
```

Comment:
```suggestion
Config.PersistentKeys = true -- Whether keys received should be saved after server restart, or not
```

Sounds better, especially for clueless server owners